### ISSUE TITLE
Making new line on /setmotd doesn't work anymore

### ISSUE THREAD

**Issue by strahil211** [Zrips/CMI#6012]
**Description of issue:**
When you use /setmotd at some point you do "\n" in it to start a new line, whenever this is done it fails to work and throws error in console

---

**ERROR (DELETE IF YOU HAVE NO ERROR):**
```
[01:41:50 WARN]: java.lang.reflect.InvocationTargetException
[01:41:50 WARN]:        at jdk.internal.reflect.GeneratedMethodAccessor64.invoke(Unknown Source)
[01:41:50 WARN]:        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[01:41:50 WARN]:        at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[01:41:50 WARN]:        at net.Zrips.CMILib.RawMessages.RawMessageManager.send(RawMessageManager.java:130)
[01:41:50 WARN]:        at net.Zrips.CMILib.RawMessages.RawMessage.show(RawMessage.java:616)
[01:41:50 WARN]:        at net.Zrips.CMILib.RawMessages.RawMessage.show(RawMessage.java:648)
[01:41:50 WARN]:        at net.Zrips.CMILib.RawMessages.RawMessage.show(RawMessage.java:642)
[01:41:50 WARN]:        at net.Zrips.CMILib.RawMessages.RawMessage.show(RawMessage.java:699)
[01:41:50 WARN]:        at net.Zrips.CMILib.Messages.CMIMultiMessage.show(CMIMultiMessage.java:239)
[01:41:50 WARN]:        at com.Zrips.CMI.CMI.sendMessage(CMI.java:2155)
[01:41:50 WARN]:        at com.Zrips.CMI.CMI.sendMessage(CMI.java:2140)
[01:41:50 WARN]:        at com.Zrips.CMI.CMI.sendMessage(CMI.java:2136)
[01:41:50 WARN]:        at com.Zrips.CMI.CMI.sendMessage(CMI.java:2132)
[01:41:50 WARN]:        at com.Zrips.CMI.commands.list.setmotd.perform(setmotd.java:50)
[01:41:50 WARN]:        at com.Zrips.CMI.commands.CommandsHandler.onCommand(CommandsHandler.java:396)
[01:41:50 WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[01:41:50 WARN]:        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
[01:41:50 WARN]:        at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:830)
[01:41:50 WARN]:        at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:2248)
[01:41:50 WARN]:        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2059)
[01:41:50 WARN]:        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2040)
[01:41:50 WARN]:        at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46)
[01:41:50 WARN]:        at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:6)
[01:41:50 WARN]:        at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:56)
[01:41:50 WARN]:        at net.minecraft.server.TickTask.run(TickTask.java:18)
[01:41:50 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:149)
[01:41:50 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(IAsyncTaskHandlerReentrant.java:23)
[01:41:50 WARN]:        at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:1407)
[01:41:50 WARN]:        at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:190)
[01:41:50 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:122)
[01:41:50 WARN]:        at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1385)
[01:41:50 WARN]:        at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1378)
[01:41:50 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:132)
[01:41:50 WARN]:        at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1354)
[01:41:50 WARN]:        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1265)
[01:41:50 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
[01:41:50 WARN]:        at java.base/java.lang.Thread.run(Thread.java:831)
[01:41:50 WARN]: Caused by: com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Invalid escape sequence at line 1 column 365 path $[1].extra[0].text
[01:41:50 WARN]:        at com.google.gson.internal.Streams.parse(Streams.java:60)
[01:41:50 WARN]:        at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:65)
[01:41:50 WARN]:        at net.minecraft.util.ChatDeserializer.a(ChatDeserializer.java:493)
[01:41:50 WARN]:        at net.minecraft.util.ChatDeserializer.a(ChatDeserializer.java:517)
[01:41:50 WARN]:        at net.minecraft.network.chat.IChatBaseComponent$ChatSerializer.a(IChatBaseComponent.java:432)
[01:41:50 WARN]:        ... 37 more
[01:41:50 WARN]: Caused by: com.google.gson.stream.MalformedJsonException: Invalid escape sequence at line 1 column 365 path $[1].extra[0].text
[01:41:50 WARN]:        at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1559)
[01:41:50 WARN]:        at com.google.gson.stream.JsonReader.readEscapeCharacter(JsonReader.java:1550)
[01:41:50 WARN]:        at com.google.gson.stream.JsonReader.nextQuotedValue(JsonReader.java:1004)
[01:41:50 WARN]:        at com.google.gson.stream.JsonReader.nextString(JsonReader.java:815)
[01:41:50 WARN]:        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:718)
[01:41:50 WARN]:        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:739)
[01:41:50 WARN]:        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:731)
[01:41:50 WARN]:        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:739)
[01:41:50 WARN]:        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:731)
[01:41:50 WARN]:        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:714)
[01:41:50 WARN]:        at com.google.gson.internal.Streams.parse(Streams.java:48)
[01:41:50 WARN]:        ... 41 more
```

**CONFIG SECTION (DELETE IF NOT RELEVANT):**
```
#######################################
## REPLACE WITH RELEVANT CONFIG SECTION  ##
#######################################
```

---

**Cmi Version (using`/cmi version`):** 
9.0.2.4
**Server Type (Spigot/Paperspigot/etc):** 
Paper
**Server Version (using `/ver`):** 
1.17.1
**Relevant plugins (Delete if this isn't needed):**

**Comment by strahil211** [comment]
Forgot to mention a key part, I did a gradient from /colorpicker to go through top line to bottom line
{#white>} A Minecraft Server \n Online 24/7 {#corn<}
Which fails, however I went into server.properties and manually added the \n and the motd showed correctly

**Comment by strahil211** [comment]
Fixed in 9.0.2.5

Thank you Zrips!
